site/src/components/state-info/WorkElectState.tsx

```diff
@@ -18,7 +18,7 @@ const WorkElectState = ( { info, jurisdictions }: Props ) => (
       )}
       <div class="jurisdictions">
          {jurisdictions.map( ( { id, name } ) => (
-            <a href={`https://workelections.com/j/${id}/${name.split( " " ).join( "-" )}`}
+            <a href={`/jurisdiction/${id}`}
```

Comment:
linking to the local page

Reply:
If we're loading this locally now we can use a `stencil-route-link` and remove the new window, no?